apps/web/pages/org/[orgSlug]/index.ts

```diff
@@ -1 +0,0 @@
-export { getServerSideProps, default } from "@pages/team/[slug]";
```

Comment:
- renamed `.ts` to `.tsx` and copied the code from "@pages/team/[slug]" (since this is removed now)